samples/Azure.AI.DocumentTranslation/Generated/DocumentTranslationClient.cs

```diff
@@ -1416,131 +1416,110 @@ public virtual Operation<BinaryData> StartTranslation(RequestContent content, Re
             }
         }
 
-        internal HttpMessage CreateStartTranslationRequest(RequestContent content)
+        internal HttpMessage CreateGetDocumentStatusRequest(Guid id, Guid documentId)
```

Comment:
Out of curiosity - why are these methods being reordered?  It make it harder to see what the actual change is to this file.

Reply:
That is due to typo :)
Fixed.